social/pushbullet/57-pushbullet.js

```diff
@@ -511,14 +511,24 @@ module.exports = function(RED) {
                     res.send("[]");
                     return onError(err, config);
                 }
-                res.send(JSON.stringify(chans.devices));
+                res.send(JSON.stringify(filterdActiveDevices(chans.devices)));
             });
         }
         else {
             res.send("[]");
         }
     });
 
+    function filterdActiveDevices(devices){
+        var activeDevices = [];
+        for(var i=0;i<devices.length;i++){
+            if(chans.devices[i].active){

```

Comment:
you call this function with a parameter of `chans.devices` and define the parameter of the function as filterdActiveDevices(devices) so inside (line 525) you only need `if (devices[i].active) {`

Is that correct ?

Reply:
I'm mistake.
I will fix it.

Reply:
@dceejay 
fix it.